src/main/java/spireMapOverhaul/zones/divinitiesgaze/cards/GuidingLight.java

```diff
@@ -0,0 +1,54 @@
+package spireMapOverhaul.zones.divinitiesgaze.cards;
+
+import basemod.BaseMod;
+import com.megacrit.cardcrawl.actions.animations.TalkAction;
+import com.megacrit.cardcrawl.actions.unique.ArmamentsAction;
+import com.megacrit.cardcrawl.actions.unique.ExpertiseAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOCard;
+import spireMapOverhaul.util.Wiz;
+import spireMapOverhaul.zones.divinitiesgaze.DivinitiesGazeZone;
+
+public class GuidingLight extends AbstractSMOCard {
+  public static String ID = SpireAnniversary6Mod.makeID("GuidingLight");
+  public static boolean canTalk = true;
+
+  public GuidingLight() {
+    super(ID, DivinitiesGazeZone.ID,  -2, CardType.SKILL, CardRarity.SPECIAL, CardTarget.NONE, CardColor.COLORLESS);
+    this.exhaust = true;
+  }
+
+  @Override
+  public void upp() {
+    this.upgraded = true;
```

Comment:
I won't highlight them all, but upgraded is already handled by the overridden upgrade method